travelrec/scoring.py

```diff
@@ -0,0 +1,42 @@
+from concurrent.futures import ThreadPoolExecutor
+from travelrec.sparql import (
+    get_geofeatures,
+    get_activities,
+)
+
+def multithreading(func, args, workers):
+    with ThreadPoolExecutor(workers) as ex:
+        res = ex.map(func, args)
+    return list(res)
+
+class Scoring:
```

Comment:
Just define two functions in Scoring module, there is no need for introducing special class(and hence state).

Reply:
done